src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/ActivityLogRazorLogger.cs

```diff
@@ -51,7 +51,9 @@ public override void LogVerbose(string message)
             Log(__ACTIVITYLOG_ENTRYTYPE.ALE_INFORMATION, message);
         }
 
+#pragma warning disable VSTHRD100 // Avoid async void methods
```

Comment:
Can have this return `task` and then on the call sites just do `_ = Log(...)`

Reply:
My thought is to merge this PR as is, and log follow up issues for this, and other comments about the specific suppression of VSTHRD100 that I had to add to get the build to pass.

There are currently about 20 suppressions of VSTHRD100 in the razor tooling repo, and I don't consider the one I to be anything special, though it does prove the point that having the analyzer enabled will stop new violations creeping in, since I was alerted to it by the build failure.

Any objections?

Reply:
No objections, 🚢 🇮🇹